### ISSUE TITLE
Documentation: ur_description package structure

### ISSUE THREAD

**Issue by Victorlouisdg** [ros-industrial/universal_robot#539]
Hi, I was trying to understand the structure of the ur_description package (of the melodic-devel-staging branch) so I made this diagram to help. I thought it might be useful for other people so maybe we could add this to a readme file that's placed inside the ur_description folder?

![ur_description_package_structure](https://user-images.githubusercontent.com/23340946/94123974-bd02b280-fe54-11ea-8d95-82c35a378aa7.png)

**Comment by gavanderhoorn** [comment]
Thanks. This would indeed make a nice addition.

Note though that the structure may change, so we might have to update the diagram a few times.

**Comment by hariharan382** [comment]
one doubt: in universal_robot folder there are urX_moveit_config(X be some numer) is it moveit config folder or we should generate the moveit configuration file through moveit "setup_assistant"
please answer this
asap

**Comment by gavanderhoorn** [comment]
> one doubt: in universal_robot folder there are urX_moveit_config(X be some numer) is it moveit config folder or we should generate the moveit configuration file through moveit "setup_assistant"

The packages you refer to are indeed MoveIt configuration packages. I would have expected the `moveit_config` part in the name to be clear indication of that (it's customary to name MoveIt configuration pkgs for specific robots like thi).

> please answer this
> asap

Please don't do this. We're all volunteers here, helping a community in our free time.

I'm going to hide your (and my) comment, as they are off-topic for this issue. That won't remove them, only not show them by default.

**Comment by hariharan382** [comment]
then i generated a package in moveit but it is completely different from
already moveit config in your universal_robot package

**Comment by gavanderhoorn** [comment]
Yes, that's why I wrote you shouldn't do that, unless you have experience and would be able to edit the new configuration to work with the files we have here in this repository.

In any case: this is still off-topic for this issue. Please keep your discussion confined to #549.

**Comment by hariharan382** [comment]
mam please i just want to move the bot  by slightly by python node  in
rviz,just help me for that

**Comment by Simon248** [comment]
You can assign it to me.
